ffmpeg/README.md

```diff
@@ -0,0 +1,105 @@
+# ffmpeg Compilation Guide
+The following instructions are based off of the [official ffmpeg compilation guide](https://trac.ffmpeg.org/wiki/CompilationGuide).
+If problems are encountered during this guide, please refer to the official guide.
+
+This guide sets up [Oracle's VirtualBox](https://www.virtualbox.org/wiki/Downloads) VMs to compile the ffmpeg library.
+That way:
+
+1. Most environmental inconsistencies are mitigated
+1. Cleanup isn't needed.
+
+> NOTE
+>
+> I will be referring to the host machine as "host machine" and the virtual machine as "virtual machine". If no prefix is given, assume the instructions to be for the host machine.
+
+
+### Linux:
+
+1. Download the [Ubuntu 64 bit iso](https://www.poweronplatforms.com/enable-disable-hyper-v-windows-10-8/).
```

Comment:
❓ Should this link be https://www.poweronplatforms.com/enable-disable-hyper-v-windows-10-8/?